clients/src/main/java/org/oracle/okafka/common/requests/AbstractRequest.java

```diff
@@ -52,6 +52,5 @@ public ApiKeys apiKeyOKafka() {
             return apiKey;
         }
 
-        public abstract T build();
```

Comment:
Why did we  remove this ?

Reply:
because this is defined in the AbstractRequest Class of kafka which we are extending so we can call and override this function in the subclasses. No need to again write this here.